src/app/containers/MediaPlayer/index.jsx

```diff
@@ -147,4 +151,4 @@ MediaPlayerContainer.defaultProps = {
   ...emptyBlockArrayDefaultProps,
 };
 
-export default MediaPlayerContainer;
+export default withRouter(MediaPlayerContainer);
```

Comment:
Can I suggest we use the `useLocation` hook instead of the HOC here. like it's been used [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/PageHandlers/withVariant/index.jsx#L10)

Reply:
Sure thing - this is now updated